docs/sources/panels/panel-library.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Library panel"
```

Comment:
I suggest making this and the title plural.

Reply:
Done